analyzer/src/funTest/assets/projects/synthetic/spdx-project-expected-output.yml

```diff
@@ -1,7 +1,7 @@
 ---
 project:
   id: "SpdxDocumentFile::xyz:0.1.0"
-  definition_file_path: "analyzer/src/funTest/assets/projects/synthetic/spdx/project/project.spdx.yml"
+  definition_file_path: "analyzer/src/funTest/assets/projects/synthetic/spdx/project/project.spdx.yaml"
```

Comment:
Please do not change file extensions for YAML files from .yml to .yaml, the SPDX standard allows both extensions [1] although YAML spec recommends using .yaml.  The ORT team has chosen to use .yml as an extension for SPDX YAML output as .yml is more commonly used in the community and ORT itself uses several .yml config files such as .ort.yml 

[1] https://github.com/spdx/spdx-spec/blame/6e9f36f5b0becb96172e54cdcd614a685b0b5e9a/chapters/conformance.md#L62

Reply:
I completely get that. I did this because [1] would only take .yaml files and falsely assumed that was a standard. I will undo the change.

[1] https://github.com/spdx/tools-java.git